virt/virtual_machines/advanced_vm_management/virt-configuring-vgpu-passthrough.adoc

```diff
@@ -0,0 +1,23 @@
+[id="virt-configuring-vgpu-passthrough"]
+= Configuring vGPU passthrough
+include::modules/virt-document-attributes.adoc[]
+:context: virt-configuring-vgpu-passthrough
+
+toc::[]
+
+Your virtual machines can access a virtual GPU (vGPU) hardware. Assigning a vGPU to your virtual machine allows you to:
```

Comment:
"allows you to do the following:"

or similar to avoid carrying the lead-in sentence into the list (per IBM SG).

Reply:
Done.